style.css

```diff
@@ -6,53 +6,13 @@ body {
 
 .container {
   display: flex;
-  justify-content: space-evenly;
-  flex-direction: row;
+  flex-direction: column;
+  align-items: center;
```

Comment:
This CSS rule does the centering— the rest of the changeset is removing the unnecessary bits.